ods_ci/tests/Resources/Page/Components/Components.resource

```diff
@@ -84,25 +83,20 @@ Get DataScienceCluster Spec
     Should Be Equal As Integers    ${rc}    0
     RETURN    ${output}
 
-Check Image Pull Path Is Redhatio
-    [Documentation]    Check that the Deployment Image Pull Path is registry.redhat.io
-    [Arguments]    ${deployment_name}
-
-    # Skip pull path check if Deployment is in exclusion list
-    IF    $deployment_name in @{REDHATIO_PATH_CHECK_EXCLUSTION_LIST}
-        Log To Console    Skip image pull path check for Deployment ${deployment_name}
-        RETURN
```

Comment:
This needs to be brought back AFAIU. I noticed that at least `kserve-controller-manager` uses quay.io on all RHOAI builds.